content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -21,33 +21,57 @@ further_reading:
+| <span id="support-beta">Beta</span>                    |  Initial implementation. May not yet contain all features. Support for new features, bug & security fixes provided on a best-effort basis.                                    |
+| <span id="support-ga">General Availability (GA)</span> |  Full implementation of all features. Full support for new features, bug & security fixes.                                                                                    |
+| <span id="support-maintenance">Maintenance</span>      |  Full implementation of existing features. Does not receive new features. Support for bug & security fixes only.                                                              |
+| <span id="support-eol">End-of-life (EOL)</span>        |  No support.                                                                                                                                                                  |
+
+### Package versioning
+
+Datadog APM for .NET Core practices [semantic versioning][3].
+As this relates to downgrading runtime support, it implies:
+
+  - **Major version updates** (for example `1.0.0` to `2.0.0`) may change support for any runtime from [Beta](#support-beta)/[GA](#support-ga) to [Maintenance](#support-maintenance)/[Legacy](#support-legacy)/[EOL](#support-eol).
+  - **Minor version updates** (for example `1.0.0` to `1.1.0`) won't lower our level of support for one runtime but may add support for one.
+  - **Patch version updates** (for example `1.0.0` to `1.0.1`) will not change support for any runtime.
+
 ## Supported .NET Core runtimes
-The .NET Tracer supports automatic instrumentation on the following .NET Core versions. It also supports [.NET Framework][2].
 
-| Version              | Microsoft End of Life | APM Supported in 1.x | APM Supported in 2.x |
+The .NET Tracer supports automatic instrumentation on the following .NET Core versions. It also supports [.NET Framework][4].
+
+| Version              | Microsoft End of Life | Support level        | Package version      |
 | -------------------- | --------------------- | -------------------- | -------------------- |
-| .NET 6               |                       | No                   | Yes                  |
-| .NET 5               |                       | Yes                  | Yes                  |
-| .NET Core 3.1        | 12/03/2022            | Yes                  | Yes                  |
-| .NET Core 2.1        | 08/21/2021            | Yes                  | Yes                  |
+| .NET 6               |                       | [GA](#support-ga)    | latest (>= 2.0.0)    |
+| .NET 5               |                       | [GA](#support-ga)    | latest (>= 2.0.0)    |
+| .NET Core 3.1        | 12/03/2022            | [GA](#support-ga)    | latest               |
+| .NET Core 2.1        | 08/21/2021            | [GA](#support-ga)    | latest               |
```

Comment:
Now that we have a clear policy on what EOL, what do you think of creating a new table or adding to this table that clearly states that .NET Core 2.0 / 2.2 / 3.0 are End-Of-Life?